### ISSUE TITLE
Test: Multiple GitHub Account support & new auth account preference

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#229420]
Refs: https://github.com/microsoft/vscode/issues/127967 https://github.com/microsoft/vscode/issues/225943

- [x] anyOS @aiday-mar
- [x] anyOS @amunger
- [x] web @rzhao271

Complexity: 4

[Create Issue](https://github.com/microsoft/vscode/issues/new?body=Testing+%23229420%0A%0A&assignees=TylerLeonhardt)

---

### Description

In Insiders, we have enabled Multiple GitHub Account support. This means you can be logged in to 2+ accounts at the same time and use them for different things.

I really just want you to smoke test logging in to multiple things and seeing if it all makes sense.

### Step 0: Get an alt GitHub account

There's really only one way to do this... you gotta get/make another GitHub account. It's kinda annoying, but it could be useful later!

I guess you could also try to get the EMU account working... the one that's like `YourMicrosoftAlias_microsoft` (so like `tyleonha_microsoft`). That'd be a good test if you can figure out how to use it.

### Step 1: Grab some extensions

So first, let me list some extensions that use GitHub:

* Settings Sync
* Cloud Changes
* Remote Tunnels
* GitHub Pull Request
* GitHub Copilot & Chat
* GitHub Repositories
* GitHub Codespaces
* GitLens
* Live Share
* And More?!....

### Step 2: Change the preference

Also, to light this up, we have also added an ability to change the account preference for an extension. So for example, if Copilot Chat was originally given AccountA, a user can go in and change the preference to AccountB, and Copilot Chat will get an event for this change.

Now just a quick overview of where you could switch accounts... either:
* Account Menu -> An Account -> Manage Trusted Extensions -> Gear icon on one of the Extensions
* Extensions View -> Right click OR Gear icon on an extension
* An Extension Editor -> Gear icon towards the top

This should bring you to a Quick Pick that lets you set a default account (you may not have one initially, but after setting it, re-open the quick pick and it should say it's default)

It's possible that not every extension plays nice with this new behavior... not all of them implement the Authentication event properly... so if you feel like an extension isn't handling it properly, I'd love it if you could open an issue on the extension... but if you don't want to, open an issue here and I'll get to it.

#### Bonus

GitHub Copilot Chat and GitHub Copilot's preference has been locked... so when you update one, the other updates. Test that! It'll test better if you switch between an account that has a Copilot Sub and one that doesn't...

**Comment by aiday-mar** [comment]
I have been fiddling with the authentication experience on VS Code. Initially I was not sure how to sign in to two GitHub accounts simultaneously. I think it could be helpful if the account menu in the bottom left offered to sign in to a second GitHub account. I managed to wire a second account by turning on cloud changes and selecting a new account with `Sign in with GitHub`. Doing this added a second GitHub account in the account menu. From here I was able to change the GitHub account for GitHub Copilot and GitHub Copilot Chat (which are paired), but clicking on `Account Preferences` on the other extensions does not open a quick pick to pick an account.
